packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -324,3 +352,15 @@ function sanitizeTiming(name: string) {
   }
   return sanitized
 }
+
+function wrapOnNewLocation(onNewLocation: NewLocationListener): NewLocationListener {
+  return (newLocation, oldLocation) => {
+    let result
+    try {
+      result = onNewLocation(newLocation, oldLocation)
+    } catch (err) {
+      console.error('onNewLocation threw an error:', err)
```

Comment:
Just a thought: I think we should prefix all our console logs so it is easily identifiable for end users.

Reply:
This makes sense, what do you suggest as a prefix? what about `DD_RUM: ...`?

Reply:
Maybe `Datadog SDK:` would be better (more explicit, and we can use the same prefix for logs and core parts). Anyway, no need to do this in your PR :)